install_config/install/planning.adoc

```diff
@@ -0,0 +1,232 @@
+[[install-config-install-planning]]
+= Planning
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[inital-planning]]
+== Initial Planning
+
+For production environments, several factors influence installation. Consider
+the following questions as you read through the documentation:
+
+* _Which installation method do you want to use?_ The xref:environment-scenarios[Installation Methods]
+section provides some information about the quick and advanced installation
+methods.
+
+* _How many hosts do you require in the cluster?_ The xref:environment-scenarios[Environment Scenarios]
+section provides multiple examples of Single Master and Multiple Master
+configurations.
+
+* _How many pods are required in your cluster?_ The xref:sizing[Sizing Considerations]
+section provides limits for nodes and pods so you can calculate how large your
+environment needs to be.
+
+* _Is xref:../../admin_guide/high_availability.adoc#admin-guide-high-availability[high availability]
```

Comment:
I've left this link intact from the original content. However, we can change this if necessary.